projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card.jsx

```diff
@@ -17,7 +17,7 @@ const useVideoPressStats = () => {
 		data: stats,
 		isLoading,
 		isError,
-	} = useSimpleQuery( 'videopress stats', { path: REST_API_VIDEOPRESS_FEATURED_STATS } );
```

Comment:
I think we use query name in the error notice, so I would keep it human readable.

Reply:
I see we suppress them later in the code, but I think it's confusing if we use it one way and the other.

Reply:
I think I am going to move the error handling improvement to its own PR, I'll make these readable for now. I think later I am going to move these to constants so we aren't duplicating strings